docs/changelog.md

```diff
@@ -267,6 +267,8 @@ With `closest_repeat_parent` mode vertices for batch registration will be receiv
 `repeat` step as well as the end of the closest `repeat` step (for the next iteration). Any other parent `repeat` steps 
 will be ignored.
 
+See detailed `repeat` step batch processing explanation [here](https://medium.com/@porunov/janusgraph-1-0-0-batch-queries-multi-query-14abc8f66d93#b09f).
```

Comment:
Do you think it's possible to create a JanusGraph official medium account, and copy your article there?

Reply:
Makes sense. Will do that